seedlet/style-rtl.css

```diff
@@ -7,7 +7,7 @@ Description: Seedlet is a free WordPress theme. A two-column layout and classica
 Requires at least: 4.9.6
 Tested up to: 5.5
 Requires PHP: 5.6.2
-Version: 1.1.3-wpcom
+Version: 1.1.2-wpcom
```

Comment:
This should match the numbers above

Reply:
OK gotcha, these get updated whenever a change is made to them? I updated both to `1.1.3`